kadi/commands/tests/test_states.py

```diff
@@ -13,7 +13,7 @@
 try:
     fetch.get_time_range('dp_pitch')
     HAS_PITCH = True
-except:
+except Exception:
     HAS_PITCH = False
 
 
```

Comment:
do we have a policy on exceptions? I know that some linters would complain about this exception being too generic.

Reply:
Yes: https://occweb.cfa.harvard.edu/twiki/bin/view/Aspect/SkaPython#Catching_exceptions

@javierggt - you can't see that yet so I have emailed it to you.

Reply:
ok, I can't say anything about whether the exceptions caught are the right ones, so I just mentioned it.

Reply:
The truth is that I don't know offhand the exception that will get raised.  I can find it for one particular case, and maybe that might be fine idea.  But at the end there is a bit of pragmatism that the code inside tests need not be perfect and so I just did a programmatic replace of `except:` with `except Exception:`.